tutorials/run-spring-petclinic-on-app-engine-flex-cloudsql.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Run Spring Pet Clinic with CloudSQL on Google App Engine Flexible Environment
+description: Learn how to deploy Spring Boot Pet Clinic application to Google App Engine flexible environment and use CloudSQL.
+author: jishaa
+tags: App Engine Flex, CloudSQL, Spring Boot
+date_published: 2017-05-12
+---
+
+This tutorial will walk you through getting Spring Pet Clinic application up and running on App Engine Flex with CloudSQL.
```

Comment:
s/getting Spring/getting the Spring/

Also, global: Replace "App Engine Flex" with "App Engine flexible environment".